emnify/constants.py

```diff
@@ -39,13 +40,15 @@ class RequestDefaultHeadersKeys(ExtendedEnum):
     AUTHORIZATION = 'authorization'
     XEmnOriginApp = 'x-emn-origin-app'
     XEmnOriginAppVersion = 'x-emn-origin-app-version'
+    USER_AGENT = 'User-Agent'
 
 
 class RequestDefaultHeadersValues(ExtendedEnum):
     APPLICATION_JSON = 'application/json'
     BEARER_TOKEN = 'Bearer {}'
     PYTHONSDK_VERSION = EMNIFY_PACKAGE_VERSION
-    PYTHONSDK = 'PythonSDK'
+    PYTHONSDK = 'emnify PythonSDK'
+    USER_AGENT = PYTHONSDK + '/' + PYTHONSDK_VERSION + ' ('+ platform.system() + ', ' + platform.release() + ')'
```

Comment:
Consider including python version as well.

Reply:
Include requests version too

Reply:
Some example I can see:

`emnifyPythonSDK/1.0.0 requests/2.26.0 (Python 3.9; +https://github.com/emnify/emnify-python-sdk)`

Reply:
This would be the new User agent`emnifyPythonSDK/0.3.11 python-requests/2.29.0 (Python 3.13.0, Linux, 6.6.32-linuxkit)`